go/runtime/bundle/registry.go

```diff
@@ -152,6 +151,49 @@ func (r *Registry) AddManifest(manifest *Manifest, dir string) error {
 	return nil
 }
 
+// Manifests returns all known manifests.
+func (r *Registry) Manifests() []*ExplodedManifest {
+	r.mu.RLock()
+	defer r.mu.RUnlock()
+	return slices.Collect(maps.Values(r.manifests))
+}
+
+// RemoveManifest removes a manifest with provided hash.
+func (r *Registry) RemoveManifest(hash hash.Hash) bool {
+	r.logger.Info("Removing manifest from registry",
```

Comment:
```suggestion
	r.logger.Info("removing manifest",
```
I would still lock before this log, because the code is more readable and it shouldn't impact the performance.